From b5a012a82b103b6c8b59dbb13683fd0cd318f174 Mon Sep 17 00:00:00 2001
From: Vivek Goyal <vgoyal@redhat.com>
Date: Tue, 19 Aug 2014 22:02:59 -0400
Subject: [crypto] pkcs7: Missing inclusion of linux/err.h

Message-id: <1408485785-777-28-git-send-email-vgoyal@redhat.com>
Patchwork-id: 88391
O-Subject: [RHEL7.1 PATCH 27/33] PKCS#7: Missing inclusion of linux/err.h
Bugzilla: 1129779
RH-Acked-by: David Howells <dhowells@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

BZ:1129779

commit 8f3438ccea149647ad1849651d1e14c7b8b85e63
Author: David Howells <dhowells@redhat.com>
Date: Fri, 25 Jul 2014 11:33:53 +0100

crypto/asymmetric_keys/pkcs7_key_type.c needs to #include linux/err.h rather
than relying on getting it through other headers.

Without this, the powerpc allyesconfig build fails.

Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: David Howells <dhowells@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/crypto/asymmetric_keys/pkcs7_key_type.c b/crypto/asymmetric_keys/pkcs7_key_type.c
index b1797d2..24a547f 100644
--- a/crypto/asymmetric_keys/pkcs7_key_type.c
+++ b/crypto/asymmetric_keys/pkcs7_key_type.c
@@ -11,6 +11,7 @@
 
 #define pr_fmt(fmt) "PKCS7key: "fmt
 #include <linux/key.h>
+#include <linux/err.h>
 #include <linux/key-type.h>
 #include <crypto/pkcs7.h>
 #include <keys/user-type.h>
-- 
1.7.1